packages/typescript/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@frontside/typescript",
+  "version": "1.0.0",
+  "description": "Frontside Typescript",
+  "author": "Frontside Engineering <engineering@frontside.com>",
+  "dependencies": {
+    "@frontside/eslint-config": "1.1.1",
+    "@frontside/tsconfig": "1.0.0",
+    "eslint": "7.4.0",
+    "typescript": "3.9.6"
```

Comment:
We probably need `@typescript-eslint/parser` and `@typescript-eslint/something-else-whose-name-eludes-me-at-the-moment` right?

Reply:
Do we need to add them again? I thought having `@frontside/eslint-config` would cover it.

Reply:
The only specify peer dependencies, right? This is the package that would provide those peer dependencies.

Reply:
we only have `eslint` as peer dependency. `@typescript-eslint/eslint-plugin`, `@typescript-eslint/parser`, and `eslint-plugin-prefer-let` are dependencies in `eslint-config`.